README.md

```diff
@@ -37,6 +60,8 @@ We also host native packages for Linux on [Package Cloud](https://packagecloud.i
 
 EventStoreDB is written in a mixture of C#, C++ and JavaScript. It can run on Windows, Linux and macOS (using Docker) using the .NET Core runtime. However, the projections library (which uses the V8 javascript engine) contains platform specific code and it must be built for the platform on which you intend to run it.
```

Comment:
do we still have C++ , that was for the V8 engine no ? We got it out completely in current supported releases